admin/class-h5p-plugin-admin.php

```diff
@@ -120,6 +120,9 @@ private function __construct() {
     // Display admin notices
     add_action('admin_notices', array($this, 'admin_notices'));
 
+    //Spawn h5p
+    add_action('wp_ajax_create_h5p', array($this, 'spawn_h5p'));
```

Comment:
Can you follow the convention established elsewhere in this file for naming spawn_h5p? e.g. wp_ajax_foo_h5p_bar would point to the function ajax_foo_bar

Additionally, please also adhere to the convention of distinguishing between h5p (the library) and contents (instances of activities)

How about wp_ajax_create_h5p_content and ajax_create_content?